src/test/java/fr/inria/gforge/spoon/mojo/SpoonMojoTest.java

```diff
@@ -141,7 +139,7 @@ public void testSpoonThrowException() throws Exception {
 		final File dirOutputResults = new File(basedir, "target/spoon-maven-plugin");
 		assertThat(dirOutputResults).doesNotExist();
 	}
-
+	@Ignore("source folder could point to a non existent folder early, asking maven for source folders removes this.")
```

Comment:
error is "lost" because see comment. remove the testcase?

Reply:
I'd say yes